spec/index.bs

```diff
@@ -10163,187 +10164,259 @@ and {{GPUTextureUsage/STORAGE_BINDING|GPUTextureUsage.STORAGE_BINDING}} usage re
             <th>Format
             <th>{{GPUTextureSampleType}}
             <th>{{GPUTextureUsage/RENDER_ATTACHMENT}}
+            <th>multisampling
+            <th>resolve
             <th>{{GPUTextureUsage/STORAGE_BINDING}}
     </thead>
-    <tr><th class=stickyheader>8-bit per component<th><th><th>
+    <tr><th class=stickyheader>8-bit per component<th><th><th><th><th>
     <tr>
         <td>{{GPUTextureFormat/r8unorm}}
         <td>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
         <td>&checkmark;
+        <td>&checkmark;
+        <td>&checkmark;
         <td>
     <tr>
         <td>{{GPUTextureFormat/r8snorm}}
         <td>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
+        <td><!-- Vulkan -->
+        <td>&checkmark;
         <td>
-        <td>
+        <td><!-- Vulkan -->
     <tr>
         <td>{{GPUTextureFormat/r8uint}}
         <td>{{GPUTextureSampleType/"uint"}}
         <td>&checkmark;
+        <td>&checkmark;
+        <td><!-- Metal -->
         <td>
     <tr>
         <td>{{GPUTextureFormat/r8sint}}
         <td>{{GPUTextureSampleType/"sint"}}
         <td>&checkmark;
+        <td>&checkmark;
+        <td><!-- Metal -->
         <td>
     <tr>
         <td>{{GPUTextureFormat/rg8unorm}}
         <td>{{GPUTextureSampleType/"float"}},<br/>{{GPUTextureSampleType/"unfilterable-float"}}
         <td>&checkmark;
+        <td>&checkmark;
+        <td><!-- Metal -->
```

Comment:
Metal feature set tables seem to say RG8Unorm supports everything on all GPU families

Reply:
Nicely spotted!